### ISSUE TITLE
Log4j (CVE-2021-44228) vulnerability

### ISSUE THREAD

**Issue by cemusta** [gentics/mesh#1291]
Hey,

I just wanted to ask if someone has already checked this vulnerability and if it's affecting any gentics mesh versions?

more info: https://www.thehackernews.com/2021/12/extremely-critical-log4j-vulnerability.html

Best,
Cem

**Comment by npomaroli** [comment]
Hi,

Gentics Mesh does not use or depend on Log4j2, but uses [SLF4j](http://www.slf4j.org/) as logging facade for [Logback](http://logback.qos.ch/) or Log4j 1.2.
As of https://github.com/apache/logging-log4j2/pull/608 there is still a discussion, whether log4j 1.2 is also affected by the vulnerability, when using the JMSAppender, but the JMSAppender is not part of the SLF4j-Log4j libraries, so

Gentics Mesh is **not** affected by this vulnerability.

cheers,
Norbert
